quartz/components/Breadcrumbs.tsx

```diff
@@ -95,10 +100,14 @@ export default ((opts?: Partial<BreadcrumbOptions>) => {
       }
 
       // Add current file to crumb (can directly use frontmatter title)
-      crumbs.push({
-        displayName: fileData.frontmatter!.title,
-        path: "",
-      })
+      if (options.showCurrentPage) {
+        console.log("Show current page in breadcrumbs is set to: " + options.showCurrentPage)
+        console.log("Adding cumb: " + fileData.frontmatter!.title)
```

Comment:
```suggestion
```

remove extra console.log to reduce clutter

Reply:
Good idea, not sure how I missed those 🤦‍♂️